### PR TITLE
Link users spaces 2

### PR DESCRIPTION
**Pull request by lookupdaily**
IT WORKS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
-./coverage
\ No newline at end of file
+./coverage
```
Comment (davmcgregor) [inline .gitignore:1]: is this all working?

File: App/makersbnb.rb [hunk 2]
```diff
@@ -62,7 +62,8 @@ class Makersbnb < Sinatra::Base
   end
 
   post '/spaces' do
-    Space.create(name: params[:name], description: params[:description], price: params[:price], date_start: params[:date_start],
+    @user = (User.find_by id: session[:user_id])
+    @user.spaces.create(name: params[:name], description: params[:description], price: params[:price], date_start: params[:date_start],
```
Comment (davmcgregor) [inline App/makersbnb.rb:66]: Is this the correct way to make a space? We need to make an instance of the Space class, right? Let me know if this works

File: coverage/.last_run.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 100.0
+    "covered_percent": 98.42
```
Comment (davmcgregor) [inline coverage/.last_run.json]: Still above 95% so ok

File: db/migrate/20200225111846_create_spaces.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ def change
       t.integer :price
       t.datetime :date_start
       t.datetime :date_end
-      t.references :users, foreign_key: true
+      t.references :user, foreign_key: true
```
Comment (davmcgregor) [inline db/migrate/20200225111846_create_spaces.rb:9]: Good to check these changes don't affect how the databases function

**Review by davmcgregor — CHANGES_REQUESTED** [review]

File: App/makersbnb.rb (hunk 2 above)
Comment (lookupdaily) [reply App/makersbnb.rb:66]: all working

File: db/migrate/20200225111846_create_spaces.rb (hunk 4 above)
Comment (lookupdaily) [reply db/migrate/20200225111846_create_spaces.rb:9]: it works - We had a lot of dependencies on singular implementation. Personally I think singular makes more sense

**Review by davmcgregor — APPROVED** [review]

**Pull request merged**